v3/examples/client/main.go

```diff
@@ -14,14 +14,14 @@ import (
 
 func doRequest(txn *newrelic.Transaction) error {
 	req, err := http.NewRequest("GET", "http://localhost:8000/segments", nil)
-	if nil != err {
+	if err != nil {
```

Comment:
thanks for fixing this, these always bothered me

Reply:
As a matter of principle I've been fixing those as I encounter them. when doing other fixes.